source/BaselineOfAnsible/BaselineOfAnsible.class.st

```diff
@@ -6,28 +6,30 @@ Class {
 
 { #category : #baselines }
 BaselineOfAnsible >> baseline: spec [
-
 	<baseline>
 	spec
 		for: #pharo
 		do: [ self setUpDependencies: spec.
-
 			spec
-				package: 'Amqp-Protocol' with: [ spec requires: #('Amqp-Protocol-091' 'Amqp-Protocol-08') ];
-				package: 'Amqp-Protocol-Base';
-				package: 'Amqp-Protocol-08' with: [ spec requires: 'Amqp-Protocol-Base' ];
-				package: 'Amqp-Protocol-091' with: [ spec requires: 'Amqp-Protocol-Base' ];
-				package: 'Amqp-Tests' with: [ spec requires: 'Amqp-Protocol' ];
-				package: 'Amqp-Protocol-Tests' with: [ spec requires: 'Amqp-Protocol' ];
-				package: 'Amqp-Tools' with: [ spec requires: 'XMLParser-Core' ].
-
+				package: 'Ansible-Protocol-Core'
```

Comment:
`Ansible-Protocol-Core`, `Ansible-Protocol-091` and `Ansible-Protocol-08` aren't in any `Deployment` group. Probable are missing from `Deployment` or requires it's own group if the protocol versions are intended to be loaded separately.

I also found a bit estrange that Core depends on the specific protocol version packages.

Reply:
Good catch. Hmm I guess it get loaded because 08 and 091 versions of the protocol depend on Common. And Core depends on Common. Anyway, I will fix that.

On `Core` depending on the two versions, it just need one of them loaded. I will create an issue because I will need to change the docs also.

Reply:
It gets loaded because the Tests group have the required dependencies. But not Deployment... I suggest to create `Deployment-08` and `Deployment-091` or something like that, and add both as dependencies of the `Deployment` group. So the user can load one of the protocol versions or both. And yes, the docs needs to be updated also :)